grakn-test-tools/pom.xml

```diff
@@ -69,7 +69,12 @@
             <artifactId>embedded-redis</artifactId>
             <version>${redis-test.version}</version>
         </dependency>
-        
+        <dependency>
+            <groupId>com.github.zxl0714</groupId>
+            <artifactId>redis-mock</artifactId>
```

Comment:
I added this instead of replacing the other one because I'm not sure if it has the full behaviour. Sometimes we just need a mock and this works fine for that but I'm not sure if it store things for example.